spec/controllers/posts/unread_spec.rb

```diff
@@ -91,42 +91,56 @@ def setup_posts
     end
 
     it "manages board/post read time mismatches" do
```

Comment:
I don't think this changes the behavior here, does it? It just adjusts the timestamps (presumably to make the logic be less confusing)

Reply:
Yeah it *should* just timecop things to make them not happen at the same time (or manually use `at_time`), unless i messed up somewhere.